### ISSUE TITLE
Mismatch of implementation between training_openInstruct and training_trl

### ISSUE THREAD

**Issue by ZeguanXiao** [Yuancheng-Xu/GenARM#2]
In [arm_trainer.py](https://github.com/Yuancheng-Xu/GenARM/blob/824e846322fa2681c551b039822670ca4c2f399a/training_trl/arm_trainer.py#L21), it appears that the Simpo loss is being used. However, in [arm_utils.py](https://github.com/Yuancheng-Xu/GenARM/blob/824e846322fa2681c551b039822670ca4c2f399a/training_openInstruct/open_instruct/arm_utils.py#L16), the implementation seems to follow Equation 5 from the paper.

Could you please clarify: Which loss function is actually used during training?

**Comment by Yuancheng-Xu** [comment]
Simpo loss function uses normalization of the response length, which is not used in arm.

The implementation in [arm_trainer.py](https://github.com/Yuancheng-Xu/GenARM/blob/824e846322fa2681c551b039822670ca4c2f399a/training_trl/arm_trainer.py#L21) should be consistent with the paper.
